test.js

```diff
@@ -204,7 +204,7 @@ test('assign styles', t => {
 });
 
 test('assign other props', t => {
-	const el = <a href="video.mp4" id="a" download referrerPolicy="no-referrer">Download</a>;
```

Comment:
Supporting camel cased prop names makes it smoother for developers to migrate to React/Preact/etc in future. They won't have to rename any props then.

I prefer first option, but I realize that the standard is lowercased names, so it's nice to support both with literally no extra effort.